Manifest - Add stateful fractions w/ animations to manifest

cheapsteak opened the pull request:
![manifest-gif-trimmed](https://cloud.githubusercontent.com/assets/743976/17498216/f0ffd592-5d93-11e6-9de0-769ac8bf7bb0.gif)

The GIF seems to be a bit sped up, the transitions are not actually that fast
Also fixed the visual bug of the separateor `/` turning black as it transitions out

cheapsteak committed:
Add stateful fractions w/ animations to manifest

cheapsteak committed:
Separator should never be black

cheapsteak committed:
Tweak - easing

btiernay commented:
💯 That is very slick my friend.

btiernay commented:
Oh, and nice animated gif too!

btiernay commented:
Another thought I had was that with DnD, if the numbers change upon drop, maybe we could emphasize it some how.

andricDu commented:
😮 wow!

andricDu commented:
Did we want to change up the text at the top as well to make it a bit less wordy?

cheapsteak commented:
Thanks!

WRT shortening text
Yup! We'll have to first decide between this and the other PR as that will affect the content of the instructional text

WRT numbers changing
That's a great idea, do you have anything in mind?
May be cool to do an incrementing/decrementing counter, should be doable via jquery.animate on innerText and a custom step function

btiernay commented:
@cheapsteak Ya, if it is what I think you are talking about, that would be nifty!

cheapsteak committed:
Add <number-filter/>

Usage:

<number-tween value="summary[repoData.repoName].donorCount || 0" filter="$filter('number')"></number-tween>

Make sure the scope has access to the $filter

cheapsteak committed:
numberTween: animate from old value instead of 0

cheapsteak committed:
use numbertween

cheapsteak committed:
Add onTweenStart and onTweenEnd to numberTween

cheapsteak committed:
Add tweening state indicator to stateful-fraction

cheapsteak committed:
Update class of stateful-fraction when tweening

cheapsteak committed:
pp: tween numerator color change less obstrusive

cheapsteak commented:
Added <number-tween/> that tweens numbers

waiting for jenkins to build

cheapsteak committed:
Bugfix: karma test failing from numbertween

needed [] even when deps are empty

cheapsteak committed:
Manifest transition style changes

btiernay commented:
Closing based on feedback from @vferretti . Please create a new feature branch with the discussed changes.
